modules/local/coverage_tsv.nf

```diff
@@ -22,11 +22,13 @@ process COVERAGE_TSV {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    sed -i 1i"Sequence  Start   End ${prefix}_cov" ${mosdepth}
-    paste ${countbusogenes} ${mosdepth} > ${prefix}_coverage.tsv
+    cut -f 4 ${mosdepth} > temp1
+    sed -i 1i"${prefix}_cov" temp1
+    paste ${countbusogenes} temp1 > ${prefix}_coverage.tsv
```

Comment:
This is one of the few times where pipe makes sense. Writing over the same files in Nextflow makes me nervous.
Might be better to combine `sed` and `cut` with pipe and give the file a real name (not temp). Then run paste separately.